src/averageAges.js

```diff
@@ -14,12 +18,20 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
   // write code here
   // learn how to use array methods like .filter .map .some .every .find .reduce
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const men = people.filter(person => person.sex === 'm');
+
+  const menBasedCentury = century
+    ? men.filter(person => century === Math.ceil(person.died / 100))
+    : men;
+
+  return menBasedCentury.reduce(calcAvg, 0) / menBasedCentury.length;
```

Comment:
with proper `calcAvg` you just do 
```suggestion
  return calcAvg(menBasedCentury);
```
```

Reply:
and same for women